src/RelayKnownVariableNames.ts

```diff
@@ -0,0 +1,38 @@
+import { ValidationRule } from "graphql"
+import { GraphQLError } from "./dependencies"
+import { getRecursiveVariableUsagesWithRelayInfo } from "./argumentDefinitions"
+
+export function undefinedVarMessage(
+  varName: string,
+  opName: string | undefined,
+  usingFragmentName: string | null
+): string {
+  opName = !opName ? "unnamed operation" : opName
+  return usingFragmentName
+    ? `Variable "$${varName}" is not defined by operation "${opName}", it is being used by "${usingFragmentName}".`
```

Comment:
Perhaps rewording it like so is a better UX, as it will more easily surface the fragment name in situations where there is limited UI real-estate and the operation that it applies to is already pretty clear when you click on the reported problem. What do you think?

```suggestion
    ? `Variable "$${varName}" is used by fragment "${usingFragmentName}", but not defined by operation "${opName}".`
```

<img width="1652" alt="Screenshot 2019-09-10 at 12 43 46" src="https://user-images.githubusercontent.com/2320/64607375-ac59d880-d3c8-11e9-8b68-9b435ada92db.png">

Reply:
Seems reasonable, will fix